tools/mock-service-host/src/mid/coordinator.ts

```diff
@@ -66,7 +66,13 @@ export class Coordinator {
                 nearest = parseInt(code)
             }
         }
-        if (nearest) return [nearest, responses[nearest.toString()].body]
+        if (nearest)
+            return [
+                nearest,
+                responses[nearest.toString()].body === undefined
+                    ? {}
```

Comment:
Shall we check response media type before doing this? '{}' should be only a good default if media type is json

Reply:
It's a good point. But for now the example's response hasn't been paired with schema. I've added a TODO comment and will refine it during the schema pairing change.

Reply:
After offline discussion, deleted this logic.